src/main/java/nextstep/subway/applicaion/dto/LineRequest.java

```diff
@@ -26,4 +27,6 @@ public Long getDownStationId() {
     public int getDistance() {
         return distance;
     }
+
+    public int getDuration() { return duration; }
```

Comment:
```suggestion
    public int getDuration() { 
        return duration; 
    }
```

컨벤션에 맞게 작성해 주면 어떨까요? 😄